### PR TITLE
DOC-199: AudioScope JSDoc

### PR DESCRIPTION
**Pull request by ctrlaltdavid**
Namespaces:
- AudioScope

Also:
- A couple of miscellaneous JSDoc fixes, noticed in passing.

Case: https://highfidelity.atlassian.net/browse/DOC-199

Note: No executable code changes.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Can someone from High Fidelity approve this PR?

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21992/HighFidelity-Beta-PR16367-fa9bf4d.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21992/HighFidelity-Beta-Interface-PR16367-fa9bf4d.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21992/HighFidelity-Beta-PR16367-fa9bf4d.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21992/HighFidelity-Quest-Beta-PR16367-fa9bf4d.apk)

**Comment by ctrlaltdavid** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22013/HighFidelity-Beta-PR16367-fa9bf4d.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22013/HighFidelity-Quest-Beta-PR16367-fa9bf4d.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22013/HighFidelity-Beta-PR16367-fa9bf4d.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22013/HighFidelity-Beta-Interface-PR16367-fa9bf4d.dmg

**Comment by ctrlaltdavid** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22015/HighFidelity-Beta-PR16367-fa9bf4d.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22015/HighFidelity-Quest-Beta-PR16367-fa9bf4d.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22015/HighFidelity-Beta-PR16367-fa9bf4d.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22015/HighFidelity-Beta-Interface-PR16367-fa9bf4d.exe

File: interface/src/audio/AudioScope.h [hunk 1]
```diff
@@ -58,159 +61,186 @@ class AudioScope : public QObject, public Dependency {
...
 
     /**jsdoc
+     * Triggered when pause changes.
      * @function AudioScope.pauseChanged
      * @returns {Signal}
      */
     void pauseChanged();
 
     /**jsdoc
+     * Triggered when scoep is triggered.
```
Comment (jennaingersoll) [inline interface/src/audio/AudioScope.h]:
```suggestion
     * Triggered when scope is triggered.
```

Comment (ctrlaltdavid) [reply interface/src/audio/AudioScope.h]: Fixed.
And excluded AudioScope API from JSDoc output.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22080/HighFidelity-Beta-PR16367-2591aed.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22080/HighFidelity-Beta-Interface-PR16367-2591aed.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22080/HighFidelity-Beta-PR16367-2591aed.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22080/HighFidelity-Quest-Beta-PR16367-2591aed.apk)

**Comment by ctrlaltdavid** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22101/HighFidelity-Beta-PR16367-2591aed.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22101/HighFidelity-Beta-Interface-PR16367-2591aed.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22101/HighFidelity-Beta-PR16367-2591aed.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22101/HighFidelity-Beta-Interface-PR16367-2591aed.exe

**Pull request merged**